src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/EmailDomainControllerTest.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.resource
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.verifyNoInteractions
+import org.mockito.kotlin.whenever
+import uk.gov.justice.digital.hmpps.manageusersapi.service.EmailDomainService
+import java.util.UUID
+
+internal class EmailDomainControllerTest {
+
+  private val emailDomainService: EmailDomainService = mock()
+  private val emailDomains: List<EmailDomainDto> = mock()
+  private val emailDomain: EmailDomainDto = mock()
+  private val emailDomainController = EmailDomainController(emailDomainService)
+  private val createEmailDomain: CreateEmailDomainDto = mock()
+
+  @Test
+  fun domainListRetrieved() {
+    whenever(emailDomainService.domainList()).thenReturn(emailDomains)
+
+    val actual = emailDomainController.domainList()
+
+    assertEquals(emailDomains, actual)
```

Comment:
Interesting... what value actually gets used for emailDomains?  Isn't it just an empty object?  Shouldn't the emailDomains (and emailDomain) objects be objects with expected values to check?